src/Settings.php

```diff
@@ -506,12 +506,21 @@ public function __construct()
                     [
                         'name' => __('VWO A/B testing', 'planet4-master-theme-backend'),
                         'desc' => __(
-                            'Adds a Content Security Policy exception for VWO.',
+                            'Adds a Content Security Policy exception for VWO and enables its anti-flicker mechanism. Do not enable this if you are A/B testing with other tools.',
                             'planet4-master-theme-backend'
                         ),
                         'id' => 'enable_vwo',
                         'type' => 'checkbox',
                     ],
+                    [
+                        'name' => __('VWO Account ID', 'planet4-master-theme-backend'),
+                        'desc' => __(
+                            'ID provided by VWO. Please make sure the VWO Testing option above is enabled.',
+                            'planet4-master-theme-backend'
+                        ),
+                        'id' => 'vwo_account_id',
+                        'type' => 'text',
+                    ],
```

Comment:
Maybe we can have just this one setting. If it's empty, we do nothing. If there is an Account ID set here then we do both CSP and anti-flickering.

In that case we can use the `desc` text from the previous setting here to explain what will happen if they add the Accound ID. _"Adds a Content Security Policy exception for VWO and enables its anti-flicker mechanism."_